sdk/servicebus/azure-servicebus/tests/stress_tests/stress_test_base.py

```diff
@@ -57,7 +58,8 @@ def __init__(self,
                  send_delay = .01,
                  receive_delay = 0,
                  should_complete_messages = True,
-                 max_message_count = 1):
+                 max_message_count = 1,
+                 send_session_id = None):
```

Comment:
will we have session receive stress test?
I don't see session stress tests come in pair -- only sending, no receiving

Reply:
`     def test_stress_queue_long_renew_session_send_and_receive(self, servicebus_namespace_connection_string, servicebus_queue):`

:)  Gotcha covered.

(I can see how this might have been unclear though, because you pass in the receivers from the test, but session id is provided for sending at send-time, so to make a receiver stress test you simply pre-populate that receiver with the session ID, whereas with sending you'd either have to use a pre-send hook to populate the message, or have a helper parameter like this so the framework does it for you.)